packages/app/cypress/e2e/runner/ct-framework-errors.cy.ts

```diff
@@ -0,0 +1,271 @@
+import type { ProjectFixtureDir } from '@tooling/system-tests/'
```

Comment:
This file is a quick smoke test for each of our major frameworks (and major versions thereof) to validate basic error conditions work as expected: uncaught error on mount, sync error, async error, and command failure.

Svelte is currently skipped due to issue with sourcemaps, will be a follow-on task

Reply:
Is there a reason we don't have `Next` or `Nuxt` in this test?

Reply:
bump

Reply:
@ZachJW34 No particular reason; I think I got distracted making sure we had base framework coverage. Unfortunately it looks like the existing Next & Nuxt system-test projects don't extend the base React and Vue fixtures, so I would have to duplicate the `Error` components and test suites I created out of those. Happen to know of a good way to avoid that? Not complex, just annoying to duplicate that sort of code

Reply:
Seems like a smoke test for Next.js might be nice... it's our most popular framework. It also looks like there is a `project-fixtures/next`, maybe we can piggy back off that.